Formula/libxml2.rb

```diff
@@ -41,7 +41,7 @@ class Libxml2 < Formula
 
   keg_only :provided_by_osx
 
-  depends_on :python => :optional
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
> For the record, I don't have any strong feelings about @ versioning and I don't recall opposing it.

OK, good to know. Versioned formulae and system duplicate formulae are keg-only by default, FYI.